web-app/CKS.Web/Startup.cs

```diff
@@ -61,11 +58,30 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 					);
 
 			app.UseDefaultFiles();
-			app.UseStaticFiles();
-
 			app.UseRouting();
+            app.UseAuthorization();
+
+			app.UseStaticFiles(new StaticFileOptions
```

Comment:
Can you put this back next to `UseDefaultFiles` please as they're related middleware - use defaultfiles means index.html is served by `UseStaticFiles`